rclpy/rclpy/impl/__init__.py

```diff
@@ -14,12 +14,24 @@
 
 import importlib
 import os
+from pathlib import Path
 
 
 def _import(name):
     try:
         return importlib.import_module(name, package='rclpy')
     except ImportError as e:
+        if e.path is None:
+            import sysconfig
+            expected_path = Path(__file__).parents[1] / (
+                name[1:] + sysconfig.get_config_var('EXT_SUFFIX'))
+            assert not expected_path.is_file()
+            e.msg += \
+                f"\nThe C extension '{expected_path}' isn't present on the " \
```

Comment:
does dashing minimum python version support f-strings?

Reply:
Good catch. Dashing targets Python 3.5 which doesn't support f-strings. Fixed in 0262fc488bbcf24686af2eafe525b3e271f903b3.